---
title: "Curriculum Vitae"
date: 2017-12-09T18:29:00-05:00
menu: ["main", "CV"]
toc: True
---

Andrew E. Brereton
------------------

*A formally trained structural biologist with an emphasis in data
science and visualization, machine learning, and software development.
Research interests are varied but include neurobiology, human/animal
intelligence, sleep biology, aging, perception, and protein
structure/design problems.*

Peer-reviewed Publications
--------------------------

**Brereton AE**, Karplus PA, **"Ensemblator v3: Robust Atom-level
Comparative Analyses and Classification of Protein Structure
Ensembles."** *Protein Sci*. (2017). DOI:
[10.1002/pro.3249](http://onlinelibrary.wiley.com/doi/10.1002/pro.3249/full)  
    
Nelson KJ, Perkins A, Van Swearingen AED, Hartman A, **Brereton
AE**, Parsonage D, Salsbury FR, Karplus PA, and Poole LB.
**"Experimentally Dissecting the Origins of Peroxiredoxin Catalysis."**
*Antioxid Redox Signal* (2016) [DOI:
10.1089/ars.2016.6922](http://online.liebertpub.com/doi/abs/10.1089/ars.2016.6922?journalCode=ars)  
    
Sharaf NG, **Brereton AE**, Byeon IL, Karplus PA, Gronenborn AM.
**"NMR structure of the HIV-1 reverse transcriptase thumb
subdomain."** *J Biomol NMR* (2016). DOI:
[10.1007/s10858-016-0077-2](https://brereton.me/_publications/NMRStructureOfTheHIV-1ReverseTranscriptaseThumbSubdomain,JBiomolNMR.pdf)  
    
**Brereton AE**, Karplus PA, **"On the reliability of peptide
non-planarity seen in ultra-high resolution crystal structures."**
*Protein Sci*. (2016). DOI:
[10.1002/pro.2883](https://brereton.me/_publications/Brereton-planarity-ProtSci-2016.pdf)  
    
**Brereton AE**, Karplus PA, **"Native proteins trap high-energy
transit conformations."** *Sci Adv.* 1, e1501188 (2015). DOI:
[10.1126/sciadv.1501188](http://advances.sciencemag.org/content/1/9/e1501188)  

Open-Source Software Development Projects
-----------------------------

#### [The Ensemblator](http://www.github.com/atomoton/ensemblator)

Python program to generate and analyze ensembles of protein
structures. Uses a novel "core" based method for determining
optimal structural overlays. Groups for comparison can be
manually defined, or automatically detected using clustering.
Capable of comparing ensembles of non-identical proteins, using
sequence alignment before structural overlay.

#### [The Reassigner](http://www.github.com/atomoton/reassigner)

Python program capable of assigning secondary structure to a pdb
formatted protein structure. Assigns by using a combination of
results from machine learning assignments of secondary structure
based on local conformation, and a calculation of a virtual
geometry. Capable of a very large range of intelligent secondary
structure assignments.

#### [Uncomplicated I Ching](https://play.google.com/store/apps/details?id=me.brereton.iching2)

A minimal android app to enable consulting the I Ching Oracle.

Education
---------

#### Ph.D. Biochemistry & Biophysics
Oregon State University  
Corvallis, OR, USA  
2013 - 2017  

Special Topics Classes:  
Macromolecular X-ray Crystallography, 
Biocomputing: Molecular simulation, Introduction to UNIX,
Command-line data analysis, Recursion and Dynamic Programming,
Topics in Data Visualization, Data Programming in R, Genome
Expression & Regulation, Protein Evolution, Scientific Skills and
Ethics, Algorithms for Computational Molecular Biology

#### B.Sc. Honours in Molecular Biology and Genetics, with distinction 
University of Guelph  
Guelph, ON, Canada  
2009 - 2013  


Awards and Honours
------------------
Scholar's Insights (Three Minute Thesis Competition), **First Place**, Oregon State University, 2016  
DataFest Oregon, **Winner, "Best Insight"**, Oregon State University, 2016  
**Lenore Bayley Graduate Fellowship**, Oregon State University, 2016  
Gordon Research Conference: Proteins **Best Poster**, Holderness School, 2015  
CGRB 2015 Spring Conference **Lightning Talk Award**, Oregon State University, 2015  
**Entrance Scholarship**, University of Guelph, 2009  

Professional Experience
-----------------------

#### Bioinformatics Developer 
Cyclica Inc., Toronto, ON  
2017 - Present  

-   Method development
-   Package deployment using Pip
-   Maintain and document application changes
-   Scientific research and feasibility studies
-   Data science and visualization
-   AI training, testing, and development


#### Graduate Research Assistant
Oregon State University, Corvallis, OR  
2014 - 2017  
Advisor: P. Andrew Karplus  

-   Development of new metrics and software tools to compare and analyze large sets of protein structures.
-   Responsible for running and maintaining the Protein Geometry Database.
-   Conducted novel research on protein geometry, evolution, and structure.
-   Conducted novel research on the structure-function relationship of tyrosine nitration using calmodulin as a model system.
-   Conducted novel research on the evolution and history of the GULOP pseudogene.
-   X-ray crystallography data collection, structure determination, and refinement.
-   Protein expression, purification, and crystallization.
-   Use of a home-source X-ray generator, as well as safety, maintenance, and repair protocols.
-   Training undergraduate researchers and graduate students.


#### Graduate Teaching Assistant
Oregon State University, Corvallis, OR  
2013 - 2014  

- Cell & Molecular Biology - F'13, S'14
   -   Developed lesson plans and problem sets to complement professor's lectures for groups of approx. 30 students in classroom setting.
   -   Provided additional individual instruction for students needing assistance.
- Biochemistry Lab - W'14
   -   Supervised students and guided experiments during lab sessions.

#### Undergraduate Research
University of Guelph, Guelph, ON  
2012 - 2013  

-   Conducted novel research on the role of Luman in endocrine pathways and identification of its downstream targets.
-   Cultured mHypoE-44 cells, gained experience in cloning, subcloning, sequencing and sequence analysis, Western blotting, transfection, qPCR, RT-PCR, RNA and DNA purification, luciferase reporter assays, and others.
-   Gained certification for mouse handling training.

Scientific Presentations
------------------------

Brereton, A. E. "Ensemblator v3: *actually* comparing ensembles".
West Coast Protein Crystallography Workshop XXIII, Asilomar, CA.
March 19-22 2017. **Poster.**  

Brereton, A. E. "On The Reliability Of Peptide Non-Planarity or: How
I Learned To Stop Worrying And Love The Bond". Northwest
Crystallograpy Workshop. Pullman, WA. June 17-19 2016. **Talk.**  

Brereton, A. E. "Seeing is believing: mapping a protein folding
transition". Scholar's Insights (Three Minute Thesis Competition).
Oregon State University. 2016. **Talk.** *Awarded First Place.*  

Brereton, A. E. "Ramachandran's Mountain Pass: Capturing a
High-energy Transition State". Volcano Conference. Seattle WA. March
3-5 2016. **Talk.**  

Brereton, A. E. "Ramachandran's mountain pass: exploring
impossibilities". GRC: Proteins. June 16 2015. **Poster.** *Selected
for best poster.*  

Brereton, A. E. "Ramachandran's mountain pass: exploring
impossibilities". CGRB 2015 Spring Conference, Oregon State
University. April 6-7 2015. **Talk.** *Selected for best
presentation.*  

Brereton, A. E. "Ramachandran's mountain pass: exploring
impossibilities". West Coast Protein Crystallography Workshop XXII,
Monterey, CA. March 16-19 2015. **Talk.**  

Skills and Techniques (not otherwise listed)
--------------------------------------------
Programming  

- R, Shiny, ggplot2, Python 2 & 3, Jupyter, rdkit, 
    Tkinter, BASH, markdown, HTML, CSS

Experience running a web server, extremely comfortable with CLI.  

Experience with cloud computing.

Familiarity with various bioinformatic resources, including MUSCLE,
MAFFT, Clustal, genome browsers, SCOP, Uniprot, Pali Database, Dali
web server, and others.  

Experience writing large software, alone or working with groups and collaborators on large
computational projects.  

Protein purification from large tissue samples, use of expanded
genetic code for unnatural amino acid incorporation in proteins.  

Services
--------
Organizational Committee member for the 2014 Northwest
Crystallography Workshop.  

Member of the Strategic Planning Committee for the Oregon State
Department of Biochemistry & Biophysics.  

Tsoo King Memorial Lecture Series 2016 Lead Organizer

RSCB Protein Data Bank Entries
------------------------------

[4XRA](http://www.rcsb.org/pdb/explore.do?structureId=4XRA) | 1.75 Ã… |
*Salmonella typhimurium* AhpC T43S mutant (Perkins et. al, *Antioxid
Redox Signal* 2016)  

Miscellaneous
-------------
Fourth Kyu certification in Aikikai Aikido.  

Gold certification as a pharmacy technician.  

Adequate sense of humour.  


