<!DOCTYPE html>
<html lang='en'>

<head>
  <meta charset='utf-8'>
<meta name='viewport' content='width=device-width, initial-scale=1'>
<meta name='description' content='Andrew E. Brereton A formally trained structural biologist with an emphasis in data science and visualization, machine learning, and software development. Research interests are varied but include neurobiology, human/animal intelligence, sleep biology, aging, perception, and protein structure/design problems.
Peer-reviewed Publications Evangelidis T, Nerli S, Novacek J, Brereton AE, Karplus A, Dotas R, Venditti V, Sgourakis N, Tripsianes K. &ldquo;Automated NMR resonance assignments and structure determination using a minimal set of 4D spectra.'>
<meta name='theme-color' content='#ffcd00'>

<meta property='og:title' content='Curriculum Vitae • Andrew E Brereton'>
<meta property='og:description' content='Andrew E. Brereton A formally trained structural biologist with an emphasis in data science and visualization, machine learning, and software development. Research interests are varied but include neurobiology, human/animal intelligence, sleep biology, aging, perception, and protein structure/design problems.
Peer-reviewed Publications Evangelidis T, Nerli S, Novacek J, Brereton AE, Karplus A, Dotas R, Venditti V, Sgourakis N, Tripsianes K. &ldquo;Automated NMR resonance assignments and structure determination using a minimal set of 4D spectra.'>
<meta property='og:url' content='https://brereton.me/brereton-cv/'>
<meta property='og:site_name' content='Andrew E Brereton'>
<meta property='og:type' content='article'><meta property='article:section' content=''><meta property='article:published_time' content='2017-12-30T00:00:00Z'/><meta property='article:modified_time' content='2017-12-30T00:00:00Z'/><meta name='twitter:card' content='summary'>

<meta name="generator" content="Hugo 0.40.2" />

  <title>Curriculum Vitae • Andrew E Brereton</title>
  <link rel='canonical' href='https://brereton.me/brereton-cv/'>
  
  
  <link rel='icon' href='/favicon.ico'>
<link rel='stylesheet' href='/assets/css/main.b0a0b202.css'><link rel='stylesheet' href='/css/theme_overrides.css'><style>
:root{--color-accent:#ffcd00;}
</style>

<script>
window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;
ga('create', 'UA-74739470-1', 'auto');
ga('send', 'pageview');
</script>
<script async src='//www.google-analytics.com/analytics.js'></script>

</head>


<body class='page type-page'>

  <div class='site'>

    <a class='screen-reader-text' href='#content'>Skip to Content</a>

    <div class='main'>

      <nav id='main-menu' class='menu main-menu' aria-label='Main Menu'>
  <div class='container'>
    
    <ul><li class='item'>
        <a href='/'>home</a>
      </li><li class='item'>
        <a href='/blog'>blog</a>
      </li><li class='item'>
        <a href='/tags'>tags</a>
      </li><li class='item'>
        <a href='/photography'>photography</a>
      </li><li class='item'>
        <a href='/writing'>writing</a>
      </li><li class='item current'>
        <a aria-current='page' href='/brereton-cv/'>Curriculum Vitae</a>
      </li></ul>
  </div>
</nav>

      <header id='header' class='header site-header'>
        <div class='container sep-after'>
          <div class='header-info'><p class='site-title title'>Andrew E Brereton</p><p class='desc site-desc'></p>
          </div>
        </div>
      </header>

      <main id='content'>


<article lang='en' class='entry'>
  <header class='header entry-header'>
  <div class='container sep-after'>
    <div class='header-info'>
      <h1 class='title'>Curriculum Vitae</h1>
      

    </div>
    
<div class='entry-meta'>
  <span class='posted-on'><svg class='icon' viewbox='0 0 24 24' stroke-linecap='round' stroke-linejoin='round' stroke-width='2' aria-hidden='true'>
  
  <rect x="3" y="4" width="18" height="18" rx="2" ry="2"/>
  <line x1="16" y1="2" x2="16" y2="6"/>
  <line x1="8" y1="2" x2="8" y2="6"/>
  <line x1="3" y1="10" x2="21" y2="10"/>
  
</svg>
<span class='screen-reader-text'>Posted on </span>
  <time class='entry-date' datetime='2017-12-30T00:00:00Z'>2017, Dec 30</time>
</span>

  
  
<span class='reading-time'><svg class='icon' viewbox='0 0 24 24' stroke-linecap='round' stroke-linejoin='round' stroke-width='2' aria-hidden='true'>
  
  <circle cx="12" cy="12" r="10"/>
  <polyline points="12 6 12 12 15 15"/>
  
</svg>
5 mins read
</span>


</div>


  </div>
</header>

  
  
<details class='container entry-toc'>
  <summary class='title'>
    <span>Table of Contents</span>
  </summary>
  <nav id="TableOfContents">
<ul>
<li>
<ul>
<li><a href="#andrew-e-brereton">Andrew E. Brereton</a></li>
<li><a href="#peer-reviewed-publications">Peer-reviewed Publications</a></li>
<li><a href="#open-source-software-development-projects">Open-Source Software Development Projects</a>
<ul>
<li>
<ul>
<li><a href="#the-ensemblator-http-www-github-com-atomoton-ensemblator"><a href="http://www.github.com/atomoton/ensemblator">The Ensemblator</a></a></li>
<li><a href="#the-reassigner-http-www-github-com-atomoton-reassigner"><a href="http://www.github.com/atomoton/reassigner">The Reassigner</a></a></li>
<li><a href="#uncomplicated-i-ching-https-play-google-com-store-apps-details-id-me-brereton-iching2"><a href="https://play.google.com/store/apps/details?id=me.brereton.iching2">Uncomplicated I Ching</a></a></li>
<li><a href="#winnower-https-github-com-atomoton-winnower"><a href="https://github.com/atomoton/winnower">Winnower</a></a></li>
</ul></li>
</ul></li>
<li><a href="#education">Education</a>
<ul>
<li>
<ul>
<li><a href="#ph-d-biochemistry-biophysics">Ph.D. Biochemistry &amp; Biophysics</a></li>
<li><a href="#b-sc-honours-in-molecular-biology-and-genetics-with-distinction">B.Sc. Honours in Molecular Biology and Genetics, with distinction</a></li>
</ul></li>
</ul></li>
<li><a href="#awards-and-honours">Awards and Honours</a></li>
<li><a href="#professional-experience">Professional Experience</a>
<ul>
<li>
<ul>
<li><a href="#scientist">Scientist</a></li>
<li><a href="#graduate-research-assistant">Graduate Research Assistant</a></li>
<li><a href="#graduate-teaching-assistant">Graduate Teaching Assistant</a></li>
<li><a href="#undergraduate-research">Undergraduate Research</a></li>
</ul></li>
</ul></li>
<li><a href="#scientific-presentations">Scientific Presentations</a></li>
<li><a href="#skills-and-techniques-not-otherwise-listed">Skills and Techniques (not otherwise listed)</a></li>
<li><a href="#services">Services</a></li>
<li><a href="#rscb-protein-data-bank-entries">RSCB Protein Data Bank Entries</a></li>
<li><a href="#miscellaneous">Miscellaneous</a></li>
</ul></li>
</ul>
</nav>
</details>


  <div class='container entry-content'>
  

<h2 id="andrew-e-brereton">Andrew E. Brereton</h2>

<p><em>A formally trained structural biologist with an emphasis in data
science and visualization, machine learning, and software development.
Research interests are varied but include neurobiology, human/animal
intelligence, sleep biology, aging, perception, and protein
structure/design problems.</em></p>

<h2 id="peer-reviewed-publications">Peer-reviewed Publications</h2>

<p>Evangelidis T, Nerli S, Novacek J, <strong>Brereton AE</strong>, Karplus A, Dotas R,
Venditti V, Sgourakis N, Tripsianes K. <strong>&ldquo;Automated NMR resonance assignments
and structure determination using a minimal set of 4D spectra.&rdquo;</strong> <em>Nat Commun</em>.
 (2018). <a href="https://www.nature.com/articles/s41467-017-02592-z">10.1038/s41467-017-02592-z</a></p>

<p><strong>Brereton AE</strong>, Karplus PA, <strong>&ldquo;Ensemblator v3: Robust Atom-level
Comparative Analyses and Classification of Protein Structure
Ensembles.&rdquo;</strong> <em>Protein Sci</em>. (2017). DOI:
<a href="http://onlinelibrary.wiley.com/doi/10.1002/pro.3249/full">10.1002/pro.3249</a></p>

<p>Nelson KJ, Perkins A, Van Swearingen AED, Hartman A, <strong>Brereton
AE</strong>, Parsonage D, Salsbury FR, Karplus PA, and Poole LB.
<strong>&ldquo;Experimentally Dissecting the Origins of Peroxiredoxin Catalysis.&rdquo;</strong>
<em>Antioxid Redox Signal</em> (2016) <a href="http://online.liebertpub.com/doi/abs/10.1089/ars.2016.6922?journalCode=ars">DOI:
10.1089/ars.2016.6922</a></p>

<p>Sharaf NG, <strong>Brereton AE</strong>, Byeon IL, Karplus PA, Gronenborn AM.
<strong>&ldquo;NMR structure of the HIV-1 reverse transcriptase thumb
subdomain.&rdquo;</strong> <em>J Biomol NMR</em> (2016). DOI:
<a href="https://brereton.me/_publications/NMRStructureOfTheHIV-1ReverseTranscriptaseThumbSubdomain,JBiomolNMR.pdf">10.1007/s10858-016-0077-2</a></p>

<p><strong>Brereton AE</strong>, Karplus PA, <strong>&ldquo;On the reliability of peptide
non-planarity seen in ultra-high resolution crystal structures.&rdquo;</strong>
<em>Protein Sci</em>. (2016). DOI:
<a href="https://brereton.me/_publications/Brereton-planarity-ProtSci-2016.pdf">10.1002/pro.2883</a></p>

<p><strong>Brereton AE</strong>, Karplus PA, <strong>&ldquo;Native proteins trap high-energy
transit conformations.&rdquo;</strong> <em>Sci Adv.</em> 1, e1501188 (2015). DOI:
<a href="http://advances.sciencemag.org/content/1/9/e1501188">10.1126/sciadv.1501188</a></p>

<h2 id="open-source-software-development-projects">Open-Source Software Development Projects</h2>

<h4 id="the-ensemblator-http-www-github-com-atomoton-ensemblator"><a href="http://www.github.com/atomoton/ensemblator">The Ensemblator</a></h4>

<p>Python program to generate and analyze ensembles of protein
structures. Uses a novel &ldquo;core&rdquo; based method for determining
optimal structural overlays. Groups for comparison can be
manually defined, or automatically detected using clustering.
Capable of comparing ensembles of non-identical proteins, using
sequence alignment before structural overlay.</p>

<h4 id="the-reassigner-http-www-github-com-atomoton-reassigner"><a href="http://www.github.com/atomoton/reassigner">The Reassigner</a></h4>

<p>Python program capable of assigning secondary structure to a pdb
formatted protein structure. Assigns by using a combination of
results from machine learning assignments of secondary structure
based on local conformation, and a calculation of a virtual
geometry. Capable of a very large range of intelligent secondary
structure assignments.</p>

<h4 id="uncomplicated-i-ching-https-play-google-com-store-apps-details-id-me-brereton-iching2"><a href="https://play.google.com/store/apps/details?id=me.brereton.iching2">Uncomplicated I Ching</a></h4>

<p>A minimal android app to enable consulting the I Ching Oracle.</p>

<h4 id="winnower-https-github-com-atomoton-winnower"><a href="https://github.com/atomoton/winnower">Winnower</a></h4>

<p>A simplistic deep neural network implementation for automatic swiping on Tinder.</p>

<h2 id="education">Education</h2>

<h4 id="ph-d-biochemistry-biophysics">Ph.D. Biochemistry &amp; Biophysics</h4>

<p>Oregon State University<br />
Corvallis, OR, USA<br />
2013 - 2017</p>

<p>Special Topics Classes:<br />
Macromolecular X-ray Crystallography,
Biocomputing: Molecular simulation, Introduction to UNIX,
Command-line data analysis, Recursion and Dynamic Programming,
Topics in Data Visualization, Data Programming in R, Genome
Expression &amp; Regulation, Protein Evolution, Scientific Skills and
Ethics, Algorithms for Computational Molecular Biology</p>

<h4 id="b-sc-honours-in-molecular-biology-and-genetics-with-distinction">B.Sc. Honours in Molecular Biology and Genetics, with distinction</h4>

<p>University of Guelph<br />
Guelph, ON, Canada<br />
2009 - 2013</p>

<h2 id="awards-and-honours">Awards and Honours</h2>

<p>Scholar&rsquo;s Insights (Three Minute Thesis Competition), <strong>First Place</strong>, Oregon State University, 2016<br />
DataFest Oregon, <strong>Winner, &ldquo;Best Insight&rdquo;</strong>, Oregon State University, 2016<br />
<strong>Lenore Bayley Graduate Fellowship</strong>, Oregon State University, 2016<br />
Gordon Research Conference: Proteins <strong>Best Poster</strong>, Holderness School, 2015<br />
CGRB 2015 Spring Conference <strong>Lightning Talk Award</strong>, Oregon State University, 2015<br />
<strong>Entrance Scholarship</strong>, University of Guelph, 2009</p>

<h2 id="professional-experience">Professional Experience</h2>

<h4 id="scientist">Scientist</h4>

<p>Cyclica Inc., Toronto, ON<br />
2017 - Present</p>

<ul>
<li>Method development</li>
<li>Package deployment using Pip</li>
<li>Maintain and document application changes</li>
<li>Scientific research and feasibility studies</li>
<li>Data science and visualization</li>
<li>AI training, testing, and development</li>
</ul>

<h4 id="graduate-research-assistant">Graduate Research Assistant</h4>

<p>Oregon State University, Corvallis, OR<br />
2014 - 2017<br />
Advisor: P. Andrew Karplus</p>

<ul>
<li>Development of new metrics and software tools to compare and analyze large sets of protein structures.</li>
<li>Responsible for running and maintaining the Protein Geometry Database.</li>
<li>Conducted novel research on protein geometry, evolution, and structure.</li>
<li>Conducted novel research on the structure-function relationship of tyrosine nitration using calmodulin as a model system.</li>
<li>Conducted novel research on the evolution and history of the GULOP pseudogene.</li>
<li>X-ray crystallography data collection, structure determination, and refinement.</li>
<li>Protein expression, purification, and crystallization.</li>
<li>Use of a home-source X-ray generator, as well as safety, maintenance, and repair protocols.</li>
<li>Training undergraduate researchers and graduate students.</li>
</ul>

<h4 id="graduate-teaching-assistant">Graduate Teaching Assistant</h4>

<p>Oregon State University, Corvallis, OR<br />
2013 - 2014</p>

<ul>
<li>Cell &amp; Molecular Biology - F&rsquo;13, S&rsquo;14

<ul>
<li>Developed lesson plans and problem sets to complement professor&rsquo;s lectures for groups of approx. 30 students in classroom setting.</li>
<li>Provided additional individual instruction for students needing assistance.</li>
</ul></li>
<li>Biochemistry Lab - W&rsquo;14

<ul>
<li>Supervised students and guided experiments during lab sessions.</li>
</ul></li>
</ul>

<h4 id="undergraduate-research">Undergraduate Research</h4>

<p>University of Guelph, Guelph, ON<br />
2012 - 2013</p>

<ul>
<li>Conducted novel research on the role of Luman in endocrine pathways and identification of its downstream targets.</li>
<li>Cultured mHypoE-44 cells, gained experience in cloning, subcloning, sequencing and sequence analysis, Western blotting, transfection, qPCR, RT-PCR, RNA and DNA purification, luciferase reporter assays, and others.</li>
<li>Gained certification for mouse handling training.</li>
</ul>

<h2 id="scientific-presentations">Scientific Presentations</h2>

<p>Brereton, A. E. &ldquo;Ensemblator v3: <em>actually</em> comparing ensembles&rdquo;.
West Coast Protein Crystallography Workshop XXIII, Asilomar, CA.
March 19-22 2017. <strong>Poster.</strong></p>

<p>Brereton, A. E. &ldquo;On The Reliability Of Peptide Non-Planarity or: How
I Learned To Stop Worrying And Love The Bond&rdquo;. Northwest
Crystallograpy Workshop. Pullman, WA. June 17-19 2016. <strong>Talk.</strong></p>

<p>Brereton, A. E. &ldquo;Seeing is believing: mapping a protein folding
transition&rdquo;. Scholar&rsquo;s Insights (Three Minute Thesis Competition).
Oregon State University. 2016. <strong>Talk.</strong> <em>Awarded First Place.</em></p>

<p>Brereton, A. E. &ldquo;Ramachandran&rsquo;s Mountain Pass: Capturing a
High-energy Transition State&rdquo;. Volcano Conference. Seattle WA. March
3-5 2016. <strong>Talk.</strong></p>

<p>Brereton, A. E. &ldquo;Ramachandran&rsquo;s mountain pass: exploring
impossibilities&rdquo;. GRC: Proteins. June 16 2015. <strong>Poster.</strong> <em>Selected
for best poster.</em></p>

<p>Brereton, A. E. &ldquo;Ramachandran&rsquo;s mountain pass: exploring
impossibilities&rdquo;. CGRB 2015 Spring Conference, Oregon State
University. April 6-7 2015. <strong>Talk.</strong> <em>Selected for best
presentation.</em></p>

<p>Brereton, A. E. &ldquo;Ramachandran&rsquo;s mountain pass: exploring
impossibilities&rdquo;. West Coast Protein Crystallography Workshop XXII,
Monterey, CA. March 16-19 2015. <strong>Talk.</strong></p>

<h2 id="skills-and-techniques-not-otherwise-listed">Skills and Techniques (not otherwise listed)</h2>

<p>Programming</p>

<ul>
<li>R, Shiny, ggplot2, Python 2 &amp; 3, Jupyter, rdkit,
Tkinter, BASH, markdown, HTML, CSS</li>
</ul>

<p>Experience running a web server, extremely comfortable with CLI.</p>

<p>Experience with cloud computing.</p>

<p>Familiarity with various bioinformatic resources, including MUSCLE,
MAFFT, Clustal, genome browsers, SCOP, Uniprot, Pali Database, Dali
web server, and others.</p>

<p>Experience writing large software, alone or working with groups and collaborators on large
computational projects.</p>

<p>Protein purification from large tissue samples, use of expanded
genetic code for unnatural amino acid incorporation in proteins.</p>

<h2 id="services">Services</h2>

<p>Organizational Committee member for the 2014 Northwest
Crystallography Workshop.</p>

<p>Member of the Strategic Planning Committee for the Oregon State
Department of Biochemistry &amp; Biophysics.</p>

<p>Tsoo King Memorial Lecture Series 2016 Lead Organizer</p>

<h2 id="rscb-protein-data-bank-entries">RSCB Protein Data Bank Entries</h2>

<p><a href="http://www.rcsb.org/pdb/explore.do?structureId=4XRA">4XRA</a> | 1.75 Å |
<em>Salmonella typhimurium</em> AhpC T43S mutant (Perkins et. al, <em>Antioxid
Redox Signal</em> 2016)</p>

<h2 id="miscellaneous">Miscellaneous</h2>

<p>Gold certification as a pharmacy technician.</p>

<p>Adequate sense of humour.</p>

<p>Speaks Esperanto</p>

</div>

  
<footer class='entry-footer'>
  <div class='container sep-before'>
  
  </div>
</footer>


</article>




      </main>

      <footer id='footer' class='footer'>
        <div class='container sep-before'>
          <section class='widget widget-social_menu sep-after'><nav aria-label='Social Menu'>
    <ul><li>
        <a href='https://github.com/atomoton' target='_blank' rel='noopener'>
          <span class='screen-reader-text'>Open Github account in new tab</span><svg class='icon' viewbox='0 0 24 24' stroke-linecap='round' stroke-linejoin='round' stroke-width='2' aria-hidden='true'>
  
  <path d="M9 19c-5 1.5-5-2.5-7-3m14 6v-3.87a3.37 3.37 0 0 0-.94-2.61c3.14-.35 6.44-1.54 6.44-7A5.44 5.44 0 0 0 20 4.77 5.07 5.07 0 0 0 19.91 1S18.73.65 16 2.48a13.38 13.38 0 0 0-7 0C6.27.65 5.09 1 5.09 1A5.07 5.07 0 0 0 5 4.77a5.44 5.44 0 0 0-1.5 3.78c0 5.42 3.3 6.61 6.44 7A3.37 3.37 0 0 0 9 18.13V22"/>
  
</svg>
</a>
      </li><li>
        <a href='https://twitter.com/atomoton' target='_blank' rel='noopener'>
          <span class='screen-reader-text'>Open Twitter account in new tab</span><svg class='icon' viewbox='0 0 24 24' stroke-linecap='round' stroke-linejoin='round' stroke-width='2' aria-hidden='true'>
  
  <path d="M23 3a10.9 10.9 0 0 1-3.14 1.53 4.48 4.48 0 0 0-7.86 3v1A10.66 10.66 0 0 1 3 4s-4 9 5 13a11.64 11.64 0 0 1-7 2c9 5 20 0 20-11.5a4.5 4.5 0 0 0-.08-.83A7.72 7.72 0 0 0 23 3z"/>
  
</svg>
</a>
      </li><li>
        <a href='mailto:andrew@brereton.me' target='_blank' rel='noopener'>
          <span class='screen-reader-text'>Contact via Email</span><svg class='icon' viewbox='0 0 24 24' stroke-linecap='round' stroke-linejoin='round' stroke-width='2' aria-hidden='true'>
  
  <path d="M4 4h16c1.1 0 2 .9 2 2v12c0 1.1-.9 2-2 2H4c-1.1 0-2-.9-2-2V6c0-1.1.9-2 2-2z"/>
  <polyline points="22,6 12,13 2,6"/>
  
</svg>
</a>
      </li><li>
        <a href='https://gitlab.com/brereton' target='_blank' rel='noopener'>
          <span class='screen-reader-text'>Open Gitlab account in new tab</span><svg class='icon' viewbox='0 0 24 24' stroke-linecap='round' stroke-linejoin='round' stroke-width='2' aria-hidden='true'>
  
  <path d="M22.65 14.39L12 22.13 1.35 14.39a.84.84 0 0 1-.3-.94l1.22-3.78 2.44-7.51A.42.42 0 0 1 4.82 2a.43.43 0 0 1 .58 0 .42.42 0 0 1 .11.18l2.44 7.49h8.1l2.44-7.51A.42.42 0 0 1 18.6 2a.43.43 0 0 1 .58 0 .42.42 0 0 1 .11.18l2.44 7.51L23 13.45a.84.84 0 0 1-.35.94z"/>
  
</svg>
</a>
      </li><li>
        <a href='https://linkedin.com/in/andrew-brereton-935ba1149' target='_blank' rel='noopener'>
          <span class='screen-reader-text'>Open Linkedin account in new tab</span><svg class='icon' viewbox='0 0 24 24' stroke-linecap='round' stroke-linejoin='round' stroke-width='2' aria-hidden='true'>
  
  <path d="M16 8a6 6 0 0 1 6 6v7h-4v-7a2 2 0 0 0-2-2 2 2 0 0 0-2 2v7h-4v-7a6 6 0 0 1 6-6z"/>
  <rect x="2" y="9" width="4" height="12"/>
  <circle cx="4" cy="4" r="2"/>
  
</svg>
</a>
      </li></ul>
  </nav>
</section>
          <div class='copyright'>
  <p> &copy; 2017-2018 Andrew E. Brereton </p>
</div>

        </div>
      </footer>

    </div>
  </div><script>window.__public_path__='\/assets\/js\/'</script>

<script src='/assets/js/main.4d3afaaf.js'></script><script src='/js/custom.js'></script>
</body>

</html>

