<!DOCTYPE html>
<html lang='en'>

<head>
  <meta charset='utf-8'>
<meta name='viewport' content='width=device-width, initial-scale=1'>
<meta name='description' content='Andrew E. Brereton A formally trained structural biologist with an emphasis in data science and visualization, machine learning, and software development. Research interests are varied but include neurobiology, human/animal intelligence, sleep biology, aging, perception, and protein structure/design problems.
Peer-reviewed Publications Brereton AE, Karplus PA, &ldquo;Ensemblator v3: Robust Atom-level Comparative Analyses and Classification of Protein Structure Ensembles.&rdquo; Protein Sci. (2017). DOI: 10.1002/pro.3249
Nelson KJ, Perkins A, Van Swearingen AED, Hartman A, Brereton AE, Parsonage D, Salsbury FR, Karplus PA, and Poole LB.'>

<meta property='og:title' content='Curriculum Vitae • brereton.me'>
<meta property='og:description' content='Andrew E. Brereton A formally trained structural biologist with an emphasis in data science and visualization, machine learning, and software development. Research interests are varied but include neurobiology, human/animal intelligence, sleep biology, aging, perception, and protein structure/design problems.
Peer-reviewed Publications Brereton AE, Karplus PA, &ldquo;Ensemblator v3: Robust Atom-level Comparative Analyses and Classification of Protein Structure Ensembles.&rdquo; Protein Sci. (2017). DOI: 10.1002/pro.3249
Nelson KJ, Perkins A, Van Swearingen AED, Hartman A, Brereton AE, Parsonage D, Salsbury FR, Karplus PA, and Poole LB.'>
<meta property='og:url' content='https://brereton.me/brereton-cv/'>
<meta property='og:site_name' content='brereton.me'>
<meta property='og:type' content='article'><meta property='article:section' content=''><meta property='article:published_time' content='2017-12-09T18:29:00-05:00'/><meta property='article:modified_time' content='2017-12-09T18:29:00-05:00'/><meta name='twitter:card' content='summary'>

<meta name="generator" content="Hugo 0.31.1" />

  <title>Curriculum Vitae • brereton.me</title>
  <link rel='canonical' href='https://brereton.me/brereton-cv/'>
  
  
  <link rel='icon' href='/favicon.ico'>
<link rel='stylesheet' href='https://fonts.googleapis.com/css?family=Ubuntu:400,400i,700&subset=latin'>
<link rel='stylesheet' href='/assets/css/main.2a477d80.css'><link rel='stylesheet' href='/css/theme_overrides.css'>
<script>
window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;
ga('create', 'UA-74739470-1', 'auto');
ga('send', 'pageview');
</script>
<script async src='//www.google-analytics.com/analytics.js'></script>

</head>


<body class='page type-page'>
  <div class='site'>

    <a class='screen-reader' href='#main'>Skip to Content</a>

    <header id='header' class='header-container'>
      <div class='header site-header'>
        <nav id='main-menu' class='main-menu-container' aria-label='Main Menu'>
  <ul class='main-menu'>
  <li>
      <a href='/'>home</a>
    </li>
  <li>
      <a href='/blog'>blog</a>
    </li>
  <li>
      <a href='/photography'>photography</a>
    </li>
  <li>
      <a href='/writing'>writing</a>
    </li>
  <li>
      <a class='current' aria-current='page' href='/brereton-cv/'>Curriculum Vitae</a>
    </li>
  
  </ul>
</nav>

        <div class='header-info'>
          
          <p class='site-title title'>brereton.me</p>
          
          <p class='site-description subtitle'></p>
        </div>
      </div>
    </header>


<main id='main' class='main'>
  <article lang='en' class='entry'>
    <header class='header-container'>
  <div class='header entry-header'>
    <div class='header-info'>
      <h1 class='title'>Curriculum Vitae</h1>
      

    </div>
    
<div class='meta'>
  <span class='posted-on'><svg class='icon' viewbox='0 0 24 24' stroke-linecap='round' stroke-linejoin='round' stroke-width='2' aria-hidden='true' preserveAspectRatio='xMidYMid meet'>
  
  <rect x="3" y="4" width="18" height="18" rx="2" ry="2"/>
  <line x1="16" y1="2" x2="16" y2="6"/>
  <line x1="8" y1="2" x2="8" y2="6"/>
  <line x1="3" y1="10" x2="21" y2="10"/>
  
</svg>
<span class='screen-reader'>Posted on </span>
  <time class='date' datetime='2017-12-09T18:29:00-05:00'>2017, Dec 09</time>
</span>

  
  
<span class='reading-time'><svg class='icon' viewbox='0 0 24 24' stroke-linecap='round' stroke-linejoin='round' stroke-width='2' aria-hidden='true' preserveAspectRatio='xMidYMid meet'>
  
  <circle cx="12" cy="12" r="10"/>
  <polyline points="12 6 12 12 15 15"/>
  
</svg>
5 mins read
</span>


</div>


  </div>
</header>

    
    

    <div class='entry-content'>
  

<h2 id="andrew-e-brereton">Andrew E. Brereton</h2>

<p><em>A formally trained structural biologist with an emphasis in data
science and visualization, machine learning, and software development.
Research interests are varied but include neurobiology, human/animal
intelligence, sleep biology, aging, perception, and protein
structure/design problems.</em></p>

<h2 id="peer-reviewed-publications">Peer-reviewed Publications</h2>

<p><strong>Brereton AE</strong>, Karplus PA, <strong>&ldquo;Ensemblator v3: Robust Atom-level
Comparative Analyses and Classification of Protein Structure
Ensembles.&rdquo;</strong> <em>Protein Sci</em>. (2017). DOI:
<a href="http://onlinelibrary.wiley.com/doi/10.1002/pro.3249/full">10.1002/pro.3249</a></p>

<p>Nelson KJ, Perkins A, Van Swearingen AED, Hartman A, <strong>Brereton
AE</strong>, Parsonage D, Salsbury FR, Karplus PA, and Poole LB.
<strong>&ldquo;Experimentally Dissecting the Origins of Peroxiredoxin Catalysis.&rdquo;</strong>
<em>Antioxid Redox Signal</em> (2016) <a href="http://online.liebertpub.com/doi/abs/10.1089/ars.2016.6922?journalCode=ars">DOI:
10.1089/ars.2016.6922</a></p>

<p>Sharaf NG, <strong>Brereton AE</strong>, Byeon IL, Karplus PA, Gronenborn AM.
<strong>&ldquo;NMR structure of the HIV-1 reverse transcriptase thumb
subdomain.&rdquo;</strong> <em>J Biomol NMR</em> (2016). DOI:
<a href="https://brereton.me/_publications/NMRStructureOfTheHIV-1ReverseTranscriptaseThumbSubdomain,JBiomolNMR.pdf">10.1007/s10858-016-0077-2</a></p>

<p><strong>Brereton AE</strong>, Karplus PA, <strong>&ldquo;On the reliability of peptide
non-planarity seen in ultra-high resolution crystal structures.&rdquo;</strong>
<em>Protein Sci</em>. (2016). DOI:
<a href="https://brereton.me/_publications/Brereton-planarity-ProtSci-2016.pdf">10.1002/pro.2883</a></p>

<p><strong>Brereton AE</strong>, Karplus PA, <strong>&ldquo;Native proteins trap high-energy
transit conformations.&rdquo;</strong> <em>Sci Adv.</em> 1, e1501188 (2015). DOI:
<a href="http://advances.sciencemag.org/content/1/9/e1501188">10.1126/sciadv.1501188</a></p>

<h2 id="open-source-software-development-projects">Open-Source Software Development Projects</h2>

<h4 id="the-ensemblator-http-www-github-com-atomoton-ensemblator"><a href="http://www.github.com/atomoton/ensemblator">The Ensemblator</a></h4>

<p>Python program to generate and analyze ensembles of protein
structures. Uses a novel &ldquo;core&rdquo; based method for determining
optimal structural overlays. Groups for comparison can be
manually defined, or automatically detected using clustering.
Capable of comparing ensembles of non-identical proteins, using
sequence alignment before structural overlay.</p>

<h4 id="the-reassigner-http-www-github-com-atomoton-reassigner"><a href="http://www.github.com/atomoton/reassigner">The Reassigner</a></h4>

<p>Python program capable of assigning secondary structure to a pdb
formatted protein structure. Assigns by using a combination of
results from machine learning assignments of secondary structure
based on local conformation, and a calculation of a virtual
geometry. Capable of a very large range of intelligent secondary
structure assignments.</p>

<h4 id="uncomplicated-i-ching-https-play-google-com-store-apps-details-id-me-brereton-iching2"><a href="https://play.google.com/store/apps/details?id=me.brereton.iching2">Uncomplicated I Ching</a></h4>

<p>A minimal android app to enable consulting the I Ching Oracle.</p>

<h2 id="education">Education</h2>

<h4 id="ph-d-biochemistry-biophysics">Ph.D. Biochemistry &amp; Biophysics</h4>

<p>Oregon State University<br />
Corvallis, OR, USA<br />
2013 - 2017</p>

<p>Special Topics Classes:<br />
Macromolecular X-ray Crystallography,
Biocomputing: Molecular simulation, Introduction to UNIX,
Command-line data analysis, Recursion and Dynamic Programming,
Topics in Data Visualization, Data Programming in R, Genome
Expression &amp; Regulation, Protein Evolution, Scientific Skills and
Ethics, Algorithms for Computational Molecular Biology</p>

<h4 id="b-sc-honours-in-molecular-biology-and-genetics-with-distinction">B.Sc. Honours in Molecular Biology and Genetics, with distinction</h4>

<p>University of Guelph<br />
Guelph, ON, Canada<br />
2009 - 2013</p>

<h2 id="awards-and-honours">Awards and Honours</h2>

<p>Scholar&rsquo;s Insights (Three Minute Thesis Competition), <strong>First Place</strong>, Oregon State University, 2016<br />
DataFest Oregon, <strong>Winner, &ldquo;Best Insight&rdquo;</strong>, Oregon State University, 2016<br />
<strong>Lenore Bayley Graduate Fellowship</strong>, Oregon State University, 2016<br />
Gordon Research Conference: Proteins <strong>Best Poster</strong>, Holderness School, 2015<br />
CGRB 2015 Spring Conference <strong>Lightning Talk Award</strong>, Oregon State University, 2015<br />
<strong>Entrance Scholarship</strong>, University of Guelph, 2009<br />
<strong>Ontario Scholar</strong>, Parry Sound High School, 2005-2009</p>

<h2 id="professional-experience">Professional Experience</h2>

<h4 id="bioinformatics-developer">Bioinformatics Developer</h4>

<p>Cyclica Inc., Toronto, ON<br />
2017 - Present</p>

<ul>
<li>Contribute new functionalities to the pipeline</li>
<li>Maintain and document application changes</li>
<li>Scientific research and development</li>
<li>Data science and visualization</li>
</ul>

<h4 id="graduate-research-assistant">Graduate Research Assistant</h4>

<p>Oregon State University, Corvallis, OR<br />
2014 - 2017<br />
Advisor: P. Andrew Karplus</p>

<ul>
<li>Development of new metrics and software tools to compare and analyze large sets of protein structures.</li>
<li>Responsible for running and maintaining the Protein Geometry Database.</li>
<li>Conducted novel research on protein geometry, evolution, and structure.</li>
<li>Conducted novel research on the structure-function relationship of tyrosine nitration using calmodulin as a model system.</li>
<li>Conducted novel research on the evolution and history of the GULOP pseudogene.</li>
<li>X-ray crystallography data collection, structure determination, and refinement.</li>
<li>Protein expression, purification, and crystallization.</li>
<li>Use of a home-source X-ray generator, as well as safety, maintenance, and repair protocols.</li>
<li>Training undergraduate researchers and graduate students.</li>
</ul>

<h4 id="graduate-teaching-assistant">Graduate Teaching Assistant</h4>

<p>Oregon State University, Corvallis, OR<br />
2013 - 2014</p>

<ul>
<li>Cell &amp; Molecular Biology - F&rsquo;13, S&rsquo;14

<ul>
<li>Developed lesson plans and problem sets to complement professor&rsquo;s lectures for groups of approx. 30 students in classroom setting.</li>
<li>Provided additional individual instruction for students needing assistance.</li>
</ul></li>
<li>Biochemistry Lab - W&rsquo;14

<ul>
<li>Supervised students and guided experiments during lab sessions.</li>
</ul></li>
</ul>

<h4 id="undergraduate-research">Undergraduate Research</h4>

<p>University of Guelph, Guelph, ON<br />
2012 - 2013</p>

<ul>
<li>Conducted novel research on the role of Luman in endocrine pathways and identification of its downstream targets.</li>
<li>Cultured mHypoE-44 cells, gained experience in cloning, subcloning, sequencing and sequence analysis, Western blotting, transfection, qPCR, RT-PCR, RNA and DNA purification, luciferase reporter assays, and others.</li>
<li>Gained certification for mouse handling training.</li>
</ul>

<h2 id="scientific-presentations">Scientific Presentations</h2>

<p>Brereton, A. E. &ldquo;Ensemblator v3: <em>actually</em> comparing ensembles&rdquo;.
West Coast Protein Crystallography Workshop XXIII, Asilomar, CA.
March 19-22 2017. <strong>Poster.</strong></p>

<p>Brereton, A. E. &ldquo;On The Reliability Of Peptide Non-Planarity or: How
I Learned To Stop Worrying And Love The Bond&rdquo;. Northwest
Crystallograpy Workshop. Pullman, WA. June 17-19 2016. <strong>Talk.</strong></p>

<p>Brereton, A. E. &ldquo;Seeing is believing: mapping a protein folding
transition&rdquo;. Scholar&rsquo;s Insights (Three Minute Thesis Competition).
Oregon State University. 2016. <strong>Talk.</strong> <em>Awarded First Place.</em></p>

<p>Brereton, A. E. &ldquo;Ramachandran&rsquo;s Mountain Pass: Capturing a
High-energy Transition State&rdquo;. Volcano Conference. Seattle WA. March
3-5 2016. <strong>Talk.</strong></p>

<p>Brereton, A. E. &ldquo;Ramachandran&rsquo;s mountain pass: exploring
impossibilities&rdquo;. GRC: Proteins. June 16 2015. <strong>Poster.</strong> <em>Selected
for best poster.</em></p>

<p>Brereton, A. E. &ldquo;Ramachandran&rsquo;s mountain pass: exploring
impossibilities&rdquo;. CGRB 2015 Spring Conference, Oregon State
University. April 6-7 2015. <strong>Talk.</strong> <em>Selected for best
presentation.</em></p>

<p>Brereton, A. E. &ldquo;Ramachandran&rsquo;s mountain pass: exploring
impossibilities&rdquo;. West Coast Protein Crystallography Workshop XXII,
Monterey, CA. March 16-19 2015. <strong>Talk.</strong></p>

<h2 id="skills-and-techniques-not-otherwise-listed">Skills and Techniques (not otherwise listed)</h2>

<p>Programming</p>

<ul>
<li>R, Shiny, ggplot2, Python 2 &amp; 3, Jupyter, rdkit,
Tkinter, BASH, markdown, HTML, CSS
<br /></li>
</ul>

<p>GIMP photo-editing software.</p>

<p>Very familiar with Windows, Mac, and Linux systems, experience running a
web server, extremely comfortable with CLI.</p>

<p>Familiarity with various bioinformatic resources, including MUSCLE,
MAFFT, Clustal, genome browsers, SCOP, Uniprot, Pali Database, Dali
web server, and others.</p>

<p>Experience writing large software for protein
analysis, experience working with groups and collaborators on large
computational projects.</p>

<p>Protein purification from large tissue samples, use of expanded
genetic code for unnatural amino acid incorporation in proteins.</p>

<h2 id="services">Services</h2>

<p>Organizational Committee member for the 2014 Northwest
Crystallography Workshop.</p>

<p>Member of the Strategic Planning Committee for the Oregon State
Department of Biochemistry &amp; Biophysics.</p>

<p>Tsoo King Memorial Lecture Series 2016 Lead Organizer</p>

<h2 id="rscb-protein-data-bank-entries">RSCB Protein Data Bank Entries</h2>

<p><a href="http://www.rcsb.org/pdb/explore.do?structureId=4XRA">4XRA</a> | 1.75 Å |
<em>Salmonella typhimurium</em> AhpC T43S mutant (Perkins et. al, <em>Antioxid
Redox Signal</em> 2016)</p>

<h2 id="miscellaneous">Miscellaneous</h2>

<p>Fourth Kyu certification in Aikikai Aikido.</p>

<p>Gold certification as a pharmacy technician.</p>

<p>Adequate sense of humour.</p>

</div>

    
<footer class='entry-footer-container'>
  <div class='entry-footer'>
  
  </div>
</footer>


  </article>
  
  
</main>

    <footer id='footer' class='footer-container'>
      <div class='footer'>
        <div class='social-menu-container'>
  <nav aria-label='Social Menu'>
    <ul class='social-menu'><li>
        <a href='https://github.com/atomoton' target='_blank' rel='noopener'>
          <span class='screen-reader'>Open Github account in new tab</span><svg class='icon' viewbox='0 0 24 24' stroke-linecap='round' stroke-linejoin='round' stroke-width='2' aria-hidden='true' preserveAspectRatio='xMidYMid meet'>
  
  <path d="M9 19c-5 1.5-5-2.5-7-3m14 6v-3.87a3.37 3.37 0 0 0-.94-2.61c3.14-.35 6.44-1.54 6.44-7A5.44 5.44 0 0 0 20 4.77 5.07 5.07 0 0 0 19.91 1S18.73.65 16 2.48a13.38 13.38 0 0 0-7 0C6.27.65 5.09 1 5.09 1A5.07 5.07 0 0 0 5 4.77a5.44 5.44 0 0 0-1.5 3.78c0 5.42 3.3 6.61 6.44 7A3.37 3.37 0 0 0 9 18.13V22"/>
  
</svg>
</a>
      </li><li>
        <a href='https://gitlab.com/brereton' target='_blank' rel='noopener'>
          <span class='screen-reader'>Open Gitlab account in new tab</span><svg class='icon' viewbox='0 0 24 24' stroke-linecap='round' stroke-linejoin='round' stroke-width='2' aria-hidden='true' preserveAspectRatio='xMidYMid meet'>
  
  <g transform-origin='center' transform="scale(0.05)"> 
  <path stroke-width='50' d="M389 77c-3-7-13-7-15 0l-44 136H182L138 77c-2-7-12-7-15 0L57 282c-2 6 0 13 5 17l194 141 194-141c5-4 7-11 5-17z"/> 
  </g>
  
</svg>
</a>
      </li><li>
        <a href='https://facebook.com/atomoton' target='_blank' rel='noopener'>
          <span class='screen-reader'>Open Facebook account in new tab</span><svg class='icon' viewbox='0 0 24 24' stroke-linecap='round' stroke-linejoin='round' stroke-width='2' aria-hidden='true' preserveAspectRatio='xMidYMid meet'>
  
  <path d="M18 2h-3a5 5 0 0 0-5 5v3H7v4h3v8h4v-8h3l1-4h-4V7a1 1 0 0 1 1-1h3z"/>
  
</svg>
</a>
      </li><li>
        <a href='https://scifi.fyi/@atomoton' target='_blank' rel='noopener'>
          <span class='screen-reader'>Open Mastodon account in new tab</span><svg class='icon' viewbox='0 0 24 24' stroke-linecap='round' stroke-linejoin='round' stroke-width='2' aria-hidden='true' preserveAspectRatio='xMidYMid meet'>
  
  <g transform-origin='center' transform="scale(0.05)"> 
  <path stroke-width='30' d="M424 294c-6 26-47 55-94 61-24 3-48 5-74 4-42-2-75-10-75-10l1 12c5 41 41 44 75 45s64-8 64-8l2 30s-24 13-67 16c-23 1-53-1-86-10-74-19-87-98-89-177v-65c0-81 53-105 53-105 27-13 73-18 121-18h2c48 0 94 5 121 18 0 0 53 24 53 105 0 0 1 60-7 102"/>
  <path stroke-width='15' d="M368 198v99h-39v-96c0-20-8-30-25-30-19 0-29 12-29 36v52h-38v-52c0-24-10-36-29-36-17 0-25 10-25 30v96h-39v-99c0-20 5-36 15-48 11-12 25-18 42-18 20 0 35 8 45 23l10 17 10-17c10-15 25-23 45-23 17 0 31 6 42 18 10 12 15 28 15 48" fill="#000"/>
  </g>
  
</svg>
</a>
      </li><li>
        <a href='https://instagram.com/atomoton' target='_blank' rel='noopener'>
          <span class='screen-reader'>Open Instagram account in new tab</span><svg class='icon' viewbox='0 0 24 24' stroke-linecap='round' stroke-linejoin='round' stroke-width='2' aria-hidden='true' preserveAspectRatio='xMidYMid meet'>
  
  <rect x="2" y="2" width="20" height="20" rx="5" ry="5"/>
  <path d="M16 11.37A4 4 0 1 1 12.63 8 4 4 0 0 1 16 11.37z"/>
  <line x1="17.5" y1="6.5" x2="17.5" y2="6.5"/>
  
</svg>
</a>
      </li><li>
        <a href='mailto:andrew@brereton.me' target='_blank' rel='noopener'>
          <span class='screen-reader'>Contact via Email</span><svg class='icon' viewbox='0 0 24 24' stroke-linecap='round' stroke-linejoin='round' stroke-width='2' aria-hidden='true' preserveAspectRatio='xMidYMid meet'>
  
  <path d="M4 4h16c1.1 0 2 .9 2 2v12c0 1.1-.9 2-2 2H4c-1.1 0-2-.9-2-2V6c0-1.1.9-2 2-2z"/>
  <polyline points="22,6 12,13 2,6"/>
  
</svg>
</a>
      </li><li>
        <a href='https://linkedin.com/in/andrew-brereton-935ba1149' target='_blank' rel='noopener'>
          <span class='screen-reader'>Open Linkedin account in new tab</span><svg class='icon' viewbox='0 0 24 24' stroke-linecap='round' stroke-linejoin='round' stroke-width='2' aria-hidden='true' preserveAspectRatio='xMidYMid meet'>
  
  <path stroke-width="1.8" d="m5.839218,4.101561c0,1.211972 -0.974141,2.194011 -2.176459,2.194011s-2.176459,-0.982039 -2.176459,-2.194011c0,-1.211094 0.974141,-2.194011 2.176459,-2.194011s2.176459,0.982917 2.176459,2.194011zm0.017552,3.94922l-4.388022,0l0,14.04167l4.388022,0l0,-14.04167zm7.005038,0l-4.359939,0l0,14.04167l4.360816,0l0,-7.370999c0,-4.098413 5.291077,-4.433657 5.291077,0l0,7.370999l4.377491,0l0,-8.89101c0,-6.915523 -7.829986,-6.66365 -9.669445,-3.259423l0,-1.891237z"/>
  
</svg>
</a>
      </li></ul>
  </nav>
</div>
        <div class='copyright'>
  <p>
        
      

       &copy; 2017 Andrew E. Brereton 
  </p>
</div>

      </div>
    </footer>

  </div><script src='/assets/js/main.5871befd.js'></script><script src='/js/custom.js'></script></body>

</html>

